src/codegen/generators/expression_generator.rs

```diff
@@ -237,6 +237,7 @@ impl<'ink, 'b> ExpressionCodeGenerator<'ink, 'b> {
             AstStatement::HardwareAccess { .. } => {
                 Ok(ExpressionValue::RValue(self.llvm.i32_type().const_zero().into()))
             }
+            AstStatement::ParenExpression(expr) => self.generate_expression_value(expr),
```

Comment:
is this legit? somehow I have the feeling, that down here we shouldnt face a ParenExpression?
I did an experiment and I tried this:
```rust
   ...
   AstStatement::ParenExpression(expr) => {
        panic!("ParenExpression should not be used for literals");
        self.generate_expression_value(expr)
   }
```

and then I ran all codegen tests and all of them worked